<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VRPMapEditorQtClass</class>
 <widget class="QMainWindow" name="VRPMapEditorQtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VRPMapEditorQt</string>
  </property>
  <widget class="CCentralWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionImport_Xml"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOperatii">
    <property name="title">
     <string>Operatii</string>
    </property>
    <addaction name="actionGenereaza_muchii"/>
    <addaction name="actionGenereaza_rute"/>
    <addaction name="actionGaseste_nod"/>
    <addaction name="actionRute_externe"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuOperatii"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMoveMap"/>
   <addaction name="actionAddClient"/>
   <addaction name="actionAddStart"/>
   <addaction name="actionAddDepot"/>
   <addaction name="actionAddTransfer"/>
   <addaction name="actionAddConnection"/>
   <addaction name="actionErase"/>
   <addaction name="actionInf"/>
   <addaction name="actionCinf"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionGenereaza_muchii">
   <property name="text">
    <string>Genereaza muchii</string>
   </property>
  </action>
  <action name="actionGenereaza_rute">
   <property name="text">
    <string>Genereaza rute...</string>
   </property>
  </action>
  <action name="actionGaseste_nod">
   <property name="text">
    <string>Gaseste nod...</string>
   </property>
  </action>
  <action name="actionMoveMap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vrpmapeditorqt.qrc">
     <normaloff>:/VRPMapEditorQt/Resources/move.png</normaloff>:/VRPMapEditorQt/Resources/move.png</iconset>
   </property>
   <property name="text">
    <string>moveMap</string>
   </property>
  </action>
  <action name="actionAddClient">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vrpmapeditorqt.qrc">
     <normaloff>:/VRPMapEditorQt/Resources/client.png</normaloff>:/VRPMapEditorQt/Resources/client.png</iconset>
   </property>
   <property name="text">
    <string>addClient</string>
   </property>
  </action>
  <action name="actionAddStart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vrpmapeditorqt.qrc">
     <normaloff>:/VRPMapEditorQt/Resources/start.png</normaloff>:/VRPMapEditorQt/Resources/start.png</iconset>
   </property>
   <property name="text">
    <string>addStart</string>
   </property>
  </action>
  <action name="actionAddDepot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vrpmapeditorqt.qrc">
     <normaloff>:/VRPMapEditorQt/Resources/depot.png</normaloff>:/VRPMapEditorQt/Resources/depot.png</iconset>
   </property>
   <property name="text">
    <string>addDepot</string>
   </property>
  </action>
  <action name="actionAddTransfer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vrpmapeditorqt.qrc">
     <normaloff>:/VRPMapEditorQt/Resources/transfer.png</normaloff>:/VRPMapEditorQt/Resources/transfer.png</iconset>
   </property>
   <property name="text">
    <string>addTransfer</string>
   </property>
  </action>
  <action name="actionAddConnection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vrpmapeditorqt.qrc">
     <normaloff>:/VRPMapEditorQt/Resources/connection.png</normaloff>:/VRPMapEditorQt/Resources/connection.png</iconset>
   </property>
   <property name="text">
    <string>addConnection</string>
   </property>
  </action>
  <action name="actionErase">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vrpmapeditorqt.qrc">
     <normaloff>:/VRPMapEditorQt/Resources/erase.png</normaloff>:/VRPMapEditorQt/Resources/erase.png</iconset>
   </property>
   <property name="text">
    <string>erase</string>
   </property>
  </action>
  <action name="actionInf">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vrpmapeditorqt.qrc">
     <normaloff>:/VRPMapEditorQt/Resources/inf.png</normaloff>:/VRPMapEditorQt/Resources/inf.png</iconset>
   </property>
   <property name="text">
    <string>inf</string>
   </property>
  </action>
  <action name="actionCinf">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vrpmapeditorqt.qrc">
     <normaloff>:/VRPMapEditorQt/Resources/cinf.png</normaloff>:/VRPMapEditorQt/Resources/cinf.png</iconset>
   </property>
   <property name="text">
    <string>cinf</string>
   </property>
  </action>
  <action name="actionImport_Xml">
   <property name="text">
    <string>Import Xml...</string>
   </property>
  </action>
  <action name="actionRute_externe">
   <property name="text">
    <string>Rute externe...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CCentralWidget</class>
   <extends>QWidget</extends>
   <header>ccentralwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="vrpmapeditorqt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolBar</sender>
   <signal>actionTriggered(QAction*)</signal>
   <receiver>VRPMapEditorQtClass</receiver>
   <slot>ActionTriggered(QAction*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menuBar</sender>
   <signal>triggered(QAction*)</signal>
   <receiver>VRPMapEditorQtClass</receiver>
   <slot>ActionTriggered(QAction*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ActionTriggered(QAction*)</slot>
 </slots>
</ui>
